### DISCUSSION TITLE
Lesson 5, news media - dangerous example for input verification

### DISCUSSION THREAD

**Discussion by andresti** [OpenMined/courses#259] [Course: Our Privacy Opportunity]
In the video, Andrew brings up the example of including cryptographic verification inside video cameras for input verification to make sure that the video really originates from the real world.
This is similar to what already happens with printers - they print a special pattern on each page so the source printer can be identified. While it indeed provides input verification, at the same time it can break output privacy - somebody wishing to remain anonymous could be identified through which printer they used - and it is suspected as a possibility of how  Reality Winner was identified (see e.g. https://www.theatlantic.com/technology/archive/2017/06/the-mysterious-printer-code-that-could-have-led-the-fbi-to-reality-winner/529350/)
The same could happen with video cameras (can we be sure it already doesn't?), so it is something to keep in mind.

**Comment by kuronosec** [comment]
Yeah I though the same. Maybe there is a way to use zero-knowledge-proofs to show that one is the original author of a video/sound stream without giving any long term identifiable information.
